indexing-hadoop/src/test/java/io/druid/indexer/HadoopIngestionSpecTest.java

```diff
@@ -33,7 +34,11 @@
 
 public class HadoopIngestionSpecTest
 {
-  private static final ObjectMapper jsonMapper = new DefaultObjectMapper();
+  private static final ObjectMapper jsonMapper;
+  static {
+    jsonMapper = new DefaultObjectMapper();
+    jsonMapper.setInjectableValues(new InjectableValues.Std().addValue(ObjectMapper.class, jsonMapper));
```

Comment:
Injected ObjectMappers should be tagged with `@Json` or `@Smile`. Is there somewhere that is not doing this?